reo/src/reopt.jl

```diff
@@ -573,14 +573,29 @@ end
 
 
 function add_nem_constraint(m, p)
+	# NEM is SalesTier 1
+	# dvStorageToGrid is always fixed at 0.0, remove it?
```

Comment:
Given the previous conversation in which storage is only intended to meet site load, it can probably be removed here and elsewhere, e.g., the site load constraint (8a).  It might make sense to do that in a separate PR though (I can do it if you prefer).

Reply:
Yes sounds good, but given the CHP merge in the pipeline I suggest we wait to reduce the number of conflicts when we merge CHP.

Reply:
Or you could remove them from both develop and chpbeta (and parallelsubproblems)